prime-router/docker-compose.yml

```diff
@@ -12,7 +12,9 @@ services:
       - type: bind
         source: ./build
         target: /prime-data-hub-router
-    entrypoint: [/prime-data-hub-router/start_func.sh]
+    # start server regardless of apidocs setup
+    command: bash -c "/prime-data-hub-router/apidocs_setup.sh ; /prime-data-hub-router/start_func.sh"
+    # entrypoint: [/prime-data-hub-router/start_func.sh]
```

Comment:
I think we might want to stick with entrypoint here and just move what happens in the apidocs_setup.sh to the `start_func.sh`

Reply:
ok, let's piggy back the api docs provision stuff into start_func.sh